build/builder.go

```diff
@@ -693,3 +728,87 @@ func (b *Builder) writeShard(fn string, ib *zoekt.IndexBuilder) (*finishedShard,
 
 // umask holds the Umask of the current process
 var umask os.FileMode
+
+// MergeMeta updates the .meta files for the shards on disk for o.
+//
+// Note: currently we only merge the RawConfig field. If a raw config field
+// exists on disk but not in the options, we keep the on disk value.
+func MergeMeta(o *Options) error {
+	// TODO should this logic live in the zoekt pkg rather than the build pkg?
+	// Argument for build is its the only place we deal with writing to multiple
+	// shards. Argument for zoekt is it's the only place that understands meta
+	// implementation.
```

Comment:
I thought I responded to this. I can do indexserver, I think that makes sense. It already understands shard name format. Still doesn't necessarily understand meta, but that is fine for now since it will atleast be contained.